main/src/cgeo/geocaching/calculator/FormulaParser.java

```diff
@@ -199,7 +199,7 @@ public final ResultWrapper parse(@NonNull final String text, @NonNull final Geop
      */
     private static final class MinDecFormulaParser extends AbstractLatLonFormulaParser {
         //                                                        (    2    )                                      (      3      )
-        private static final String STRING_MINDEC = "\\s*(" + COORD_FORMULA_PATTERN_STRING + ")[°\\s]+(" + COORD_FORMULA_PATTERN_STRING + "\\." + COORD_FORMULA_PATTERN_STRING + ")['′\\s]?";
+        private static final String STRING_MINDEC = "\\s*(" + COORD_FORMULA_PATTERN_STRING + ")[°\\s]+(" + COORD_FORMULA_PATTERN_STRING + "\\." + COORD_FORMULA_PATTERN_STRING + ")['′]?[\\s]?";
```

Comment:
Should the last part of the regExpr be `[\\s]*` instead of `[\\s]? `?

Reply:
What do you want to cover?

Square brackets meen caracter classes. And question mark an optional sign. 
See eg. https://www.hosteurope.de/blog/regex-grundlagen-arbeiten-mit-regulaeren-ausdruecken/.

Reply:
I think there is no need to restrict the last whitespace to be only zero or one.